### DISCUSSION TITLE
Uploaded proof already in use.

### DISCUSSION THREAD

**Discussion by deepakA18** [community/community#112210] [GitHub Education]
### Select Topic Area

Question

### Body

I was applying for student developer pack, I had another github account with college email id, but I deleted it as it was not in use by me.  
Then I added the college email id in my regular github account and tried to apply for the student developer pack.
When I uploaded the id proof that is my college id card it says that id proof has already been used. 
I might have used it in my deleted account, but it is deleted now, then how come this message appears ????

Please help, I want to use Copilot to assist me in my work!

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @deepakA18! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
